flipt-python/tests/conftest.py

```diff
@@ -31,3 +31,13 @@ def sync_flipt_client(flipt_url, flipt_auth_token):
 @pytest.fixture
 def async_flipt_client(flipt_url, flipt_auth_token):
     return AsyncFliptClient(url=flipt_url, authentication=ClientTokenAuthentication(flipt_auth_token))
+
+
+@pytest.fixture(scope="session")
+def sync_k8s_flipt_client(flipt_url, flipt_auth_token):
+    return FliptClient(url=flipt_url, authentication=KubernetesAuthentication(flipt_auth_token))
```

Comment:
As per above comments, we're changing this argument to be the url that Flipt is on:
```suggestion
    return FliptClient(url=flipt_url, authentication=KubernetesAuthentication(url=flipt_url))
```